askama_derive/src/config.rs

```diff
@@ -373,7 +373,7 @@ static DEFAULT_ESCAPERS: &[(&[&str], &str)] = &[
         ],
         "Html",
     ),
-    (&["md", "none", "txt", "yml", ""], "Text"),
+    (&["md", "none", "txt", "typ", "yml", ""], "Text"),
```

Comment:
On the contrary, I would say that we should remove `md` and `yml` from the list.